### PR TITLE
Add class to generate pdf

### PR DESCRIPTION
**Pull request by niphyle**
Added a class to generate a pdf and manipulate the document to add the necessary text fields

### CODE REVIEW

**Comment by Coxswain-Navigator** [comment]
Will review this after testing

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+   "liveServer.settings.port": 5501
+}
```
Comment (Coxswain-Navigator) [inline .vscode/settings.json:3]: I think this needs a trailing newline

File: index.html [hunk 2]
```diff
@@ -0,0 +1,436 @@
...
+   </main>
+   <footer>
+      This form is a demo for a new Employee Development Funds application process for Valencia College.
+      Created by: Natasha Phyle, Nicolo Perrelli, Adam Hunt, Craig Mackey, Mariele Felipe Pinheiro Rosa, Fernando
+      Jiminez, Alex Visco
+   </footer>
+</body>
+
+
+</html>
```
Comment (Coxswain-Navigator) [inline index.html:436]: another newline here

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    <dependencies>
+        <dependency>
+            <groupId>com.itextpdf</groupId>
+            <artifactId>itext7-core</artifactId>
+            <version>7.0.4</version>
+            <type>pom</type>
+        </dependency>
+    </dependencies>
+
+</project>
```
Comment (Coxswain-Navigator) [inline pom.xml:25]: and here as well

**Comment by niphyle** [comment]
The branch is ready to be merged.

**Review by Coxswain-Navigator — APPROVED** [review]
Approved.

**Pull request merged**